src/lib/mixer_module/mixer_module.cpp

```diff
@@ -159,6 +159,9 @@ bool MixingOutput::updateSubscriptions(bool allow_wq_switch)
 			// TODO: this might need to be configurable depending on the module
 			_interface.ScheduleOnInterval(100_ms);
 		}
+
+		// prevent excessive back to back execution
+		_interface.set_minimum_interval_us(250_us);
```

Comment:
A problem I see with this is that you can skip rate controller updates: if for instance RC gets updated right before the rate controller, that update is skipped.

I prefer a solution that only subscribes to the rate controller output, with a timeout/backup schedule.

Reply:
> A problem I see with this is that you can skip rate controller updates: if for instance RC gets updated right before the rate controller, that update is skipped.

Yes I'm aware of that possibility and considered this less bad overall.

> I prefer a solution that only subscribes to the rate controller output, with a timeout/backup schedule.

Likewise, but I was looking for something we could do quickly. Given the other issues I'll close it and hopefully we can find time soon to dig into the schedule of output modules soon.

Thanks